README.rst

```diff
@@ -30,14 +30,15 @@ Do not forget to change CHANGELOG.md.
 Quick start
 -----------
 
-Add ``artapi`` to your ``INSTALLED_APPS`` setting::
+client = CoreApiClient(api_url, client_id, client_id, api_version)
 
-    INSTALLED_APPS = (
+client.send_poa(dada)
 
-        'artapi',
-
-    )
+By default, django-artapi will not store the token and will have to be requested at each client initialization.
```

Comment:
It is no an accurate description. You are talking about specific situation which is not for all users.
Describe the args and users will choose their way to use the lib.